ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/storage/PrivateStateKeyValueStorage.java

```diff
@@ -39,24 +38,14 @@
   private static final Bytes METADATA_KEY_SUFFIX = Bytes.of("METADATA".getBytes(UTF_8));
   private static final Bytes STATUS_KEY_SUFFIX = Bytes.of("STATUS".getBytes(UTF_8));
   private static final Bytes REVERT_KEY_SUFFIX = Bytes.of("REVERT".getBytes(UTF_8));
+  private static final Bytes PRIVACY_GROUP_HEAD_BLOCK_MAP_PREFIX = Bytes.of("MAP".getBytes(UTF_8));
```

Comment:
`s/PRIVACY_GROUP_HEAD_BLOCK_MAP_PREFIX/PRIVACY_GROUP_HEAD_BLOCK_MAP_SUFFIX`

Also, I wonder if we should use something a bit more descriptive than just `MAP`. Maybe `PGHEADMAP`?

Reply:
sounds good